CMakeLists.txt

```diff
@@ -52,6 +52,22 @@ if (NOT BUILD_NUMBER)
   set(BUILD_NUMBER "0")
 endif ()
 
+if (UNIX)
+  option(BUILD_ENCLAVES "Build Enclaves" ON)
```

Comment:
This should not be an `option` on Unix. That implies that it can _optionally_ be turned off. But on Unix, that makes no sense. It should just be `set(BUILD_ENCLAVES ON)` (maybe with a comment to that effect).

Reply:
Good point. Fixed.